SS2-Project/Assets/Starstorm2/Modules/Pickups/Equipments/WhiteFlag.cs

```diff
@@ -2,18 +2,23 @@
 using MSU.Config;
 using RoR2;
 using RoR2.ContentManagement;
+using RoR2.Skills;
 using System.Collections;
 using System.Collections.Generic;
 using UnityEngine;
+
 namespace SS2.Equipments
 {
-    public sealed class WhiteFlag : SS2Equipment
+    public sealed class WhiteFlag : SS2Equipment, IContentPackModifier
     {
         private const string token = "SS2_EQUIP_WHITEFLAG_DESC";
         public override NullableRef<List<GameObject>> ItemDisplayPrefabs => throw new System.NotImplementedException();
```

Comment:
Replace NotImplementedException with null. As a word of thumb, never leave NotImplementedExceptions because an exception will be thrown as soon as its tried to be accessed.